### PR TITLE
DCOS-22174: Fix CI Pull Request Creation

### PR DESCRIPTION
**Pull request by juliangieseke**
PR creation failed due to a bad multiline bash command… :( this should fix that issue and also automatically adds a comment to set the created PR to "ready for review".
Also I added some minor fixes I stumbled upon while working in the files…

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -173,6 +195,12 @@ pipeline {
           sh "./scripts/ci/release-version"
         }
       }
+      post {
+        always {
+          archiveArtifacts 'pr.json'
```
Comment (DanielMSchmidt) [inline Jenkinsfile]: Why does this need to be archived?

Comment (juliangieseke) [reply Jenkinsfile]: it helps debugging errors when something went wrong while updating github.

**Comment by juliangieseke** [comment]
Still red 😩

**Comment by nLight** [comment]
There's no retry button in the pipeline for the PR. How do we re-trigger the build?

**Comment by DanielMSchmidt** [comment]
Close and open the PR again?

**Comment by juliangieseke** [comment]
yeah… jenkins renamed the jobs because I changed the config to reduce load (disabled `pr-head`). in this case you have to remove the `-merge` suffix from the jobs name and then you're anble to retrigger it -> did it in this case 👍

**Comment by juliangieseke** [comment]
```
hudson.plugins.git.GitException: Command "git checkout -f 26f22698a2f76a6b13d5207c3edf99e714ee3ed1" returned status code 128:
stdout: 
stderr: fatal: reference is not a tree: 26f22698a2f76a6b13d5207c3edf99e714ee3ed1
```
🤯 🤬

**Comment by juliangieseke** [comment]
we're going to rethink the whole 'automatic release' thing, DC/OS is going to change the way we have to do our bumps… 

I'll create a new PR wrapping up the CI optimizations without release stuff, from that point I'll start over migrating the ci/cd stuff to the new DC/OS way…

**Pull request closed without merging**